aspnetcore/blazor/tooling.md

```diff
@@ -448,13 +431,278 @@ The following example from the [tutorial for using SignalR with a Blazor WebAsse
+
+:::moniker-end
+
+:::zone pivot="vs"
+
+Press <kbd>Ctrl</kbd>+<kbd>F5</kbd> on the keyboard to run the app without the debugger.
+
+Visual Studio displays the following dialog when a project isn't configured to use SSL:
+
+![Trust self-signed certificate dialog](~/blazor/tooling/_static/trust-certificate.png)
+
+Select **Yes** if you trust the ASP.NET Core SSL certificate.
+
+The following dialog is displayed:
+
+![Security warning dialog](~/blazor/tooling/_static/install-certificate.png)
+
+Select **Yes** to acknowledge the risk and install the certificate.
+
+For information on trusting the development certificate for the Firefox browser, see <xref:security/enforcing-ssl#trust-the-https-certificate-with-firefox-to-prevent-sec_error_inadequate_key_usage-error>.
+
+Visual Studio:
+
+* Compiles and runs the app.
+* Launches the default browser at `https://localhost:{PORT}`, which displays the app's UI. The `{PORT}` placeholder is the random port assigned to the app when the app is created. If you need to change the port due to a local port conflict, you may do so in the project's `Properties/launchSettings.json` file.
+
+:::zone-end
+
+:::zone pivot="vsc"
+
+For information on trusting the HTTPS certificate for browsers other than Firefox, see the [HTTPS development certificate trust guidance](xref:security/enforcing-ssl#trust-the-aspnet-core-https-development-certificate-on-windows-and-macos). When using the Firefox browser, see the [certificate trust guidance for Firefox](xref:security/enforcing-ssl#trust-the-https-certificate-with-firefox-to-prevent-sec_error_inadequate_key_usage-error).
+
+In VS Code, press <kbd>Ctrl</kbd>+<kbd>F5</kbd> (Windows) or <kbd>⌘</kbd>+<kbd>F5</kbd> (macOS) to run the app without debugging.
+
+At the **Select debugger** prompt in the **Command Palette** at the top of the VS Code UI, select **C#**. At the next prompt, select the HTTPS profile (`C#: BlazorWebAppMovies [https]`).
+
+The default browser is launched at `https://localhost:{PORT}`, which displays the app's UI. The `{PORT}` placeholder is the random port assigned to the app when the app is created. If you need to change the port due to a local port conflict, you may do so in the project's `Properties/launchSettings.json` file.
+
+:::zone-end
+
+:::zone pivot="cli"
+
+For information on trusting the HTTPS certificate for browsers other than Firefox, see the [HTTPS development certificate trust guidance](xref:security/enforcing-ssl#trust-the-aspnet-core-https-development-certificate-on-windows-and-macos). When using the Firefox browser, see the [certificate trust guidance for Firefox](xref:security/enforcing-ssl#trust-the-https-certificate-with-firefox-to-prevent-sec_error_inadequate_key_usage-error) section of that article.
+
+In a command shell opened to the project's root folder, execute the [`dotnet run`](/dotnet/core/tools/dotnet-run) command to compile and start the app:
```

Comment:
Should we suggest `dotnet watch` instead so that you get hot reload support? I believe `dotnet watch` will also auto launch the browser on Windows (not sure on macOS or Linux).

Reply:
I suppose it was just that way for historical reasons. I'll change it here and open a follow-up issue for instances elsewhere around the docs.

Reply:
***Done!*** 👍 ... Will be on the next commit.